{"event": "verbose", "uuid": "340c61a8-f9fd-40a7-8fc9-d0712461ea47", "counter": 146, "stdout": "\u001b[0;31mERROR! Invalid extra vars data supplied. '@/Users/devops/data/project/python/pgoops/upload/iac/test6/env/extravars' could not be made into a dictionary\u001b[0m", "start_line": 145, "end_line": 146, "runner_ident": "b0e98a5d-7e40-4bfc-a5a0-8ac047898853", "created": "2022-04-21T15:40:09.890234"}